src/main/java/edu/ucsb/cs156/happiercows/jobs/MilkTheCowsJob.java

```diff
@@ -92,6 +98,14 @@ public void milkCows(JobContext ctx, Commons commons, UserCommons userCommons) {
      * @return
      */
     public static double calculateMilkingProfit(Commons commons, UserCommons userCommons) {
+        // Because this is a private method, we trust that the
+        // caller has already checked that the user is not hidden
+
+        /*
+        if (userCommons.getUser().isHidden()) {
+            return 0.0;
+        }
+        */
```

Comment:
Tell me more about this?   We still have commented out code here.  Are we not sure how to go about this?